frontend/src/app/shared/components/header/header.component.sass

```diff
@@ -0,0 +1,141 @@
+@import "src/styles/colors"
+
+@font-face
+    font-family: Barlow
+    src: url('~src/assets/fonts/Barlow-Medium.ttf')
+
+.header-wrp
+    position: fixed
+    top: 0
+    background: $white
+    color: $black
+    padding: 0px
+    box-shadow: 0px 2px 4px 2px rgba(71, 83, 108, 0.06)
+    box-sizing: border-box
+    width: 100%
+    height: 80px
+    z-index: 100
+    font-family: Barlow, serif
+
+.header
+    display: flex
+    justify-content: space-between
+    padding: 0px 57px 0px 63px
+    height: 100% 
+    width: 100%
+
+.header-content
+    display: flex
+    justify-content: center
+    flex-direction: row
+    align-items: center
+    height: 100%
+    &:focus
+        outline: none
+
+.title img
+    height: 35px
+
+.avatar-size
+    height: 36px
+    border-radius: 50%
+
+.title img
+    padding-right: 10px
+
+.title
+    font-weight: 700
+    font-size: 18px
+    line-height: 146%
+    &:hover
+      cursor: pointer
+
+.subscription
+    background: $main-color
+    color: $white
+    width: 155px
+    height: 34px
+    border: none
+    border-radius: 17px
+    font-weight: 700
+    font-size: 14px
+    line-height: 146%
+    &:hover
+      cursor: pointer
+
+.main-area p
+    margin: 0px 10px
+    font-weight: 500
+    font-size: 16px
+    line-height: 150%
+    &:hover
+      cursor: pointer
+
+.active
+    color: $main-color
+    border-bottom: 2px solid
+
+.disabled
+    color: $gray-300
+
+.remaindings
+    display: flex
+
+.reminding
+    background: $gray-100
+    color: red
+    border-radius: 50px
+    height: 36px
+    position: relative
+    width: 36px
+    margin: 0 10px
+
+.reminding img
+    margin: 0
+    position: absolute
+    top: 50%
+    left: 50%
+    -ms-transform: translate(-50%, -50%)
+    transform: translate(-50%, -50%)
+    &:hover
+      cursor: pointer
+
+.user-area
+    display: flex
+    align-items: center
+    padding-left: 20px
+    &:hover
+      cursor: pointer
+
+.user-area span
+    padding: 0 10px
+    font-weight: 500
+    font-size: 16px
+    line-height: 150%
+
+.menu-element
+    display: flex
+    justify-content: flex-start
+    align-items: center
+
+.menu-icon
+    padding-right: 10px
+
+.menu-text
+    font-family: Barlow, serif
+    font-weight: 400
+    font-size: 14px
+    line-height: 146%
+
+.log-out
+    color: $main-color
+    font-weight: 700
+    font-size: 14px
+    line-height: 146%
+
+.mat-badge-content
+    background: $time-color
+    color: $main-color
+
+p 
+  margin: 0px
```

Comment:
No global styles in local css + we should have it in some PR

Reply:
removed it